packages/kit/types/config.d.ts

```diff
@@ -22,7 +22,13 @@ export type AdapterUtils = {
 
 export type Adapter = {
 	name: string;
-	adapt: (utils: AdapterUtils) => Promise<void>;
+	adapt: ({
+		utils,
+		svelteConfig
+	}: {
+		utils: AdapterUtils;
+		svelteConfig: ValidatedConfig;
```

Comment:
maybe if we add some documentation here explaining that it's the `svelte.config.js` file then we can use the shorter `config` name?

Reply:
Docs as in JSDoc in the type definition of in the Adapter Author docs which are yet to exist?

Reply:
I'm not quite sure I understood, but I was thinking something like:

```
export type AdapterInput = {
	utils: AdapterUtils;
	/**
	 * The configuration provided by the user in svelte.config.js
	 */
	config: ValidatedConfig;
};

export type Adapter = {
	name: string;
	adapt: (input: AdapterInput) => Promise<void>;
};
```

Although now I'm wondering if that's even necessary because we have documented the type as `ValidatedConfig` which should be sufficient

Reply:
I was asking if that type of comment you suggest will be recognised and thus show in editor highlighting alongside other type information.

>Although now I'm wondering if that's even necessary because we have documented the type as `ValidatedConfig` which should be sufficient

With the adapter getting the `kit.adapter` func params and `utils`, I feel `config` is not enough information, even with the ValidatedConfig type. Will rename so we can discuss.